src/WebHandlers.cpp

```diff
@@ -58,18 +58,15 @@ AsyncStaticWebHandler& AsyncStaticWebHandler::setCacheControl(const char* cache_
 }
 
 AsyncStaticWebHandler& AsyncStaticWebHandler::setLastModified(const char* last_modified){
-  _last_modified = String(last_modified);
+  _last_modified = last_modified;
   return *this;
 }
 
-AsyncStaticWebHandler& AsyncStaticWebHandler::setLastModified(struct tm* last_modified){
-  auto formatP = PSTR("%a, %d %b %Y %H:%M:%S %Z");
-  char format[strlen_P(formatP) + 1];
-  strcpy_P(format, formatP);
-
-  char result[30];
-  strftime(result, sizeof(result), format, last_modified);
-  return setLastModified((const char *)result);
+AsyncStaticWebHandler& AsyncStaticWebHandler::setLastModified(const std::tm* last_modified){
+  constexpr size_t buffsize = sizeof("Fri, 27 Jan 2023 15:50:27 GMT");    // a format for LM header
+  char result[buffsize];
+  std::strftime(result, buffsize, "%a, %d %b %Y %H:%M:%S GMT", last_modified);
```

Comment:
- You know the length, no need for constexpr
- no behaviour change please, so keep `%Z`

```c++
char result[30];
strftime(result, 30, "%a, %d %b %Y %H:%M:%S %Z", last_modified);
```